packages/zoltan2/test/color/coloring1.cpp

```diff
@@ -148,24 +184,39 @@ int main(int narg, char** arg)
 {
   std::string inputFile = "";            // Matrix Market file to read
   std::string outputFile = "";           // Output file to write
+  std::string colorAlg = "SerialGreedy"; // Default algorithm is the serial greedy one.
   bool verbose = false;                  // Verbosity of output
+  std::string prepartition = "";         // Call Zoltan2 partitioning to better 
+                                         // distribute the 
+                                         // graph before coloring
+  bool prepartition_rows = false;        // When prepartition=rows, 
+                                         // balance wrt rows;
+  bool prepartition_nonzeros = false;    // When prepartition=nonzeros,
+                                         // balance wrt matrix nonzeros.
   int testReturn = 0;
+  int totalColors = 0;
+  int localColors = 0;
 
   ////// Establish session.
   Tpetra::ScopeGuard tscope(&narg, &arg);
   Teuchos::RCP<const Teuchos::Comm<int> > comm = Tpetra::getDefaultComm();
   int me = comm->getRank();
 
+  if (me == 0) std::cout << "Starting everything" << std::endl;
   // Read run-time options.
   Teuchos::CommandLineProcessor cmdp (false, false);
+  cmdp.setOption("colorMethod", &colorAlg,
+                 "Coloring algorithms supported: SerialGreedy, Hybrid, 2GL");
```

Comment:
I think 2GL is a variation of Hybrid, as it also uses hybrid MPI+Kokkos. Perhaps it should be called Hybrid2GL? (Optional, just a suggestion.)

Reply:
This is a good suggestion. The current name is very long, and it doesn't indicate that it follows the Hybrid method fairly closely. I'll make this change.